firmware/boron-ins-fsm/src/BraveSensorProductionFirmware.ino

```diff
@@ -23,8 +23,6 @@ void setup() {
     // enable reset reason
     System.enableFeature(FEATURE_RESET_INFO);
 
-    // use internal antenna on Boron for BLE
-    BLE.selectAntenna(BleAntennaType::INTERNAL);
```

Comment:
I have vague recollections about there being a timing issue with selecting the antenna. I wonder whether it needs to be selected before we run the `setupIM()` function? Or whether it's something that can actually be changed after the initial setup function is done. I'm going to try to see if I can find anything written down about this, but I wonder whether this should be left in so that there is at least the internal antenna when the devices powers up and then hopefully it will be changed when it hits the first run of  `loop`.

Reply:
No, I can't find anything. And it's possible that my recollections were related to the wifi antenna that we used to have on the Argons. 

So nothing to do about this comment other than be extra cautious to make sure that this works before deploying it to everyone.